README.md

```diff
@@ -13,19 +13,21 @@ This GNU Radio module is developed to realize signal analysis abilities in typic
 - OFDM parameter estimation (carrier spacing, symbol time)
 - Blind OFDM synchronization
 - Resampling of signals to constant rate
-- 3D Visualisation of FAM data, from gr-specest
-- Using TensorFlow models for AMC
+- 3D Visualisation of FAM data, from gr-specest (not on GR 3.8 yet)
+- Using TensorFlow models for AMC (not on GR 3.8 yet)
 
 ## Installation
 
 To install the Inspector, the following dependencies are required:
 
-- [GNU Radio](https://github.com/gnuradio/gnuradio) 3.7.9.3
-- gr-analog, gr-blocks, gr-channel, gr-filter
-- QT4
-- QWT 6.1.0
-- QwtPlot3D
-- TensorFlow 0.12 ([pip installation](https://www.tensorflow.org/get_started/os_setup#pip_installation) - you need the version for Python 2.7)
+- [GNU Radio](https://github.com/gnuradio/gnuradio) 3.8
+    - gr-blocks, gr-filter, gr-fft
+    - Optional for tests: gr-analog, gr-channel
+- Qt5
+- Qwt 6.1.0
+
+There is a deprecated version of gr-inspector for GNU Radio 3.7 on the maint-3.7
```

Comment:
Hm no. Bugfixes are an option, but I wanted to make clear no feature development is happening there. What wording would you propose?

Reply:
I'd rather explicit rather than implicit. A gist of the intended message would be as follows (but not necessarily should limit to it)

"For continuous use of gr-inspector with GR 3.7, please use the maint-3.7 branch where eventual bugfixes will take place, which also applies for maint-3.8 and GR3.8 respectively. Further feature development will take place in the master branch, compatible with GR 3.9, in accordance with the [OOT Migration guide](https://wiki.gnuradio.org/index.php/GNU_Radio_3.8_OOT_Module_Porting_Guide)"